### PR TITLE
Transport Permit Details

### PR DESCRIPTION
**Pull request by cameron-eyds**
*Issue #:* /bcgov/entity#18350

*Description of changes:*
* Updated Handling when Transport Permit Data exists
* Transport Permit Details component and implementation
* Misc small items and unit tests

Screenshot 2024-02-05 at 1 38 06 PM

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the PPR license (Apache 2.0).

### CODE REVIEW

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-assets-dev--pr-1721-lycrjtw2.web.app

File: ppr-ui/tests/unit/TransportPermitDetails.spec.ts [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+import flushPromises from 'flush-promises'
+
+const store = useStore()
+
+describe('Transport Permit Details', () => {
+  let wrapper
+
+  beforeEach(async () => {
+    store.setMhrInformation({
+      permitDateTime: '2024-02-05T08:40:53-08:00',
```
Comment (dimak1) [inline ppr-ui/tests/unit/TransportPermitDetails.spec.ts:16]: Wouldn't it start to fail after the date has passed?

**Review by dimak1 — APPROVED** [review]
Looks good!

File: ppr-ui/src/components/mhrTransportPermit/TransportPermitDetails.vue [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+        <h3>Transport Permit Details</h3>
+      </v-col>
+    </v-row>
+
+    <v-row
+      noGutters
+      class="pt-4"
+    >
+      <v-col cols="3">
+        <h3>Transport Permit<br> Number</h3>
```
Comment (dimak1) [inline ppr-ui/src/components/mhrTransportPermit/TransportPermitDetails.vue:17]: Not sure about new line break, maybe let it wrap by itself?

File: ppr-ui/tests/unit/TransportPermitDetails.spec.ts (hunk 1 above)
Comment (cameron-eyds) [reply ppr-ui/tests/unit/TransportPermitDetails.spec.ts:16]: This is just hard corded mock data, will always be the same.
Intention was to test for the formatted output in the template.

File: ppr-ui/src/components/mhrTransportPermit/TransportPermitDetails.vue (hunk 2 above)
Comment (cameron-eyds) [reply ppr-ui/src/components/mhrTransportPermit/TransportPermitDetails.vue:17]: We could but design wanted a break. It was either this or apply a bunch of padding to the h3.

Comment (dimak1) [reply ppr-ui/src/components/mhrTransportPermit/TransportPermitDetails.vue:17]: Ok sounds good, thanks for clarification.

**Review by doug-lovett — APPROVED** [review]
Looks good.

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-assets-dev--pr-1721-lycrjtw2.web.app

**Pull request merged**